components/application-registry/internal/metadata/specification/assetstore/service.go

```diff
@@ -0,0 +1,200 @@
+package assetstore
+
+import (
+	"crypto/tls"
+	"fmt"
+	"github.com/kyma-project/kyma/components/application-registry/internal/apperrors"
+	"github.com/kyma-project/kyma/components/application-registry/internal/metadata/specification/assetstore/docstopic"
+	"github.com/kyma-project/kyma/components/application-registry/internal/metadata/specification/assetstore/upload"
+	"github.com/kyma-project/kyma/components/application-registry/internal/metadata/specification/download"
+	"net/http"
+	"strings"
+	"time"
+)
+
+const (
+	docTopicDisplayNameFormat = "Documentation topic for service class id=%s"
+	docTopicDescriptionFormat = "Documentation topic for service class id=%s"
+	specRequestTimeout        = time.Duration(5 * time.Second)
```

Comment:
Wouldn't parametrisation of this argument be a good idea? What do you think? If we agree on that we could make this as a follow-up task.

Reply:
Same case for `components/application-registry/internal/metadata/specification/service.go`